src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionService.java

```diff
@@ -222,7 +222,7 @@ protected void setDescriptorsAndPoints(WriteFinalDecisionTemplateBodyBuilder bui
         }
 
         List<String> mobilityAnswers = ActivityType.MOBILITY.getAnswersExtractor().apply(caseData);
-        if (mobilityAnswers != null) {
+        if (mobilityAnswers != null && !"notConsidered".equals(caseData.getPipWriteFinalDecisionMobilityQuestion())) {
```

Comment:
Can you use the AwardType.NOT_CONSIDERED.getKey() here instead of hardcoded string?